src/radar.cpp

```diff
@@ -71,6 +74,30 @@ static iV_Image		radarBitmap;
 static UDWORD		*radarOverlayBuffer = nullptr;
 static Vector3i		playerpos = {0, 0, 0};
 
+class RadarWidget : public WIDGET {
+public:
+	RadarWidget();
+
+	void run(W_CONTEXT *) override;
+	void display(int xOffset, int yOffset) override;
+	bool hitTest(int x, int y) override;
+	void clicked(W_CONTEXT *, WIDGET_KEY = WKEY_PRIMARY) override;
+	void released(W_CONTEXT *, WIDGET_KEY = WKEY_PRIMARY) override;
+	void highlight(W_CONTEXT *) override;
+	void highlightLost() override;
+
+public:
+	bool hasHighlight() const { return m_hasHighlight; }
+	bool isRadarDragging() const { return m_handlingDrag; }
+
+private:
+	std::array<optional<Vector2i>, 3> clickPosition;
```

Comment:
Let's prepend `m_` at the beginning, since it's a private member, so that member names are consistent with each other.